yeat/bandage/bandage.py

```diff
@@ -0,0 +1,55 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+from pathlib import Path
+import platform
+from pkg_resources import resource_filename
+from snakemake import snakemake
+import warnings
+
+
+bandage = {
+    "Linux": Path("~/Bandage/Bandage").expanduser(),
+    "macOS": Path("~/Bandage/Bandage.app/Contents/MacOS/Bandage").expanduser(),
+}
```

Comment:
If a user wants to do a custom installation, they're welcome to install the program into a custom directory and set their `$PATH` accordingly. But by default we should assume that the `Bandage` executable is in the environment `$PATH`.

Reply:
UPDATE: Dropped this idea. Making users update their $PATH accordingly to enable the Bandage command for their OS.